Automattic/wp-calypso
# Help Center: Support Experience to 50% of ALL users

## Proposed Changes

This removes the experiment and shows the new Support Experience to 50% of **ALL** users, so now including wp-admin and the editor.

## Testing Instructions

* Checkout branch
* `yarn start` and test it in Calypso.
* Sandbox Public API, Widget API, and your site
* Sync to sandbox ( `cd apps/help-center` -> `yarn dev --sync`)
* Verify the HC flag, `flags=help-center-experience`, is still working on Calypso, WP-Admin, and Editor.
* Remove the flag from your query params
* You should try with a user whose last two digits are less than 75. You should see the new experience.
* You should not be seeing the new experience otherwise.
* Test atomic site.
